ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -380,21 +378,19 @@ public partial class PrincipiaPluginAdapter
     if (!(vessel.situation == Vessel.Situations.SUB_ORBITAL ||
           vessel.situation == Vessel.Situations.ORBITING ||
           vessel.situation == Vessel.Situations.ESCAPING ||
-          (vessel.situation == Vessel.Situations.FLYING &&
-           (vessel.altitude > manageability_altitude_threshold ||
-            vessel.altitude == -vessel.mainBody.Radius)))) {
```

Comment:
Shouldn't we keep the `-Radius` condition?  Do we want to manage the vessel at the centre of the celestial.

Reply:
As discussed, this is fine (🔥🐶☕🔥).